HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -2929,8 +2952,12 @@ def self.apply_hvac_sizing(hpxml_bldg, weather, cfa)
         if not (htg_sys.is_a?(HPXML::HeatingSystem) &&
                 [HPXML::HVACTypeBoiler,
                  HPXML::HVACTypeElectricResistance].include?(htg_sys.heating_system_type))
-          htg_sys.heating_airflow_cfm = hvac_sizing_values.Heat_Airflow.round
-          htg_sys.heating_airflow_cfm_isdefaulted = true
+          if !htg_sys.heating_airflow_cfm.nil? && hpxml_bldg.header.use_maximum_airflow_rates
+            htg_sys.heating_airflow_cfm = [hvac_sizing_values.Heat_Airflow.round, htg_sys.heating_airflow_cfm].min
```

Comment:
Should this happen in `hvac_sizing.rb` somewhere? Before `apply_hvac_installation_quality` is called?